Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -1083,6 +1083,12 @@ theorem finset_card_const_le_le_of_tsum_le {ι : Type*} {a : ι → ℝ≥0∞}
     _ ≤ c := tsum_le_c
 #align ennreal.finset_card_const_le_le_of_tsum_le ENNReal.finset_card_const_le_le_of_tsum_le
 
+theorem tsum_fiberwise (f : β → ENNReal) (g : β → γ) :
+    ∑' (x : γ), ∑' (b : (g ⁻¹' {x})), f b = ∑' i : β, f i :=
+  HasSum.tsum_eq <|
+  (((Equiv.sigmaFiberEquiv g).hasSum_iff).mpr (Summable.hasSum ENNReal.summable)).sigma <|
+  (fun _ ↦ ((ENNReal.summable).hasSum_iff).mpr rfl)
```

Comment:
Term mode proofs tend to be much harder to read than tactic proofs, where it is easier to read the order of arguments and the elementary steps. So, as a general rule, we avoid term mode for proofs which are longer than one line, although this is not an absolute rule. Do you think you could try to convert your proof to tactic mode?

Reply:
Will do!